api/v2/openapi.yaml

```diff
@@ -523,12 +523,12 @@ definitions:
         type: string
       sendResolved:
         type: boolean
-      lastNotify:
+      lastAttempt:
         type: string
         format: date-time
-      lastNotifyDuration:
+      lastAttemptDuration:
```

Comment:
```suggestion
      lastAttemptDuration:
           description: duration of the last attempt to use the integration
```

Reply:
@yuri-tceretian I just added descriptions for the fields and generated the files.